src/NServiceBus.Core/Unicast/Config/UnicastBus.cs

```diff
@@ -159,17 +145,6 @@ void RegisterMessageOwnersAndBusAddress(FeatureConfigurationContext context, IEn
                 return;
             }
 
-            if (!string.IsNullOrWhiteSpace(unicastConfig.ForwardReceivedMessagesTo))
```

Comment:
this code seems to be removed without replacement? Can you give some background on that change?